### DISCUSSION TITLE
Do I have to put my scripts in "~/.kenv/scripts", can I put them in subdirectories? like '~/.kenv/scripts/chatgpt/'

### DISCUSSION THREAD

**Discussion by alkene5** [johnlindquist/kit#1280] [Q&A] [answered]
As my scripts grew, I found it difficult to manage them all in the root directory,
But I put the scripts in the subdirectory and I can't find them in 'cmd +; `

**Comment by johnlindquist** [comment, marked as answer]
Organizing scripts is done by using "Kit Environments" AKA "kenvs".

You can see the pattern used by `~/.kenv/kenvs/examples/scripts`.

This is a requirement due to how the file watching/compiler works.

![CleanShot 2023-06-16 at 00 38 40](https://github.com/johnlindquist/kit/assets/36073/6484ff2e-93a3-465a-afde-d36ccb548a5d)

**Reply by alkene5** [reply]
ok
